src/pypromice/process/resample.py

```diff
@@ -34,13 +34,13 @@ def resample_dataset(ds_h, t):
     df_d = ds_h.to_dataframe().resample(t).mean()
     
     # recalculating wind direction from averaged directional wind speeds
-    for var in ['wdir_u','wdir_l','wdir_i']:
+    for var in ['wdir_u','wdir_l']:
         if var in df_d.columns:
             if ('wspd_x_'+var.split('_')[1] in df_d.columns) & ('wspd_x_'+var.split('_')[1] in df_d.columns):
                 df_d[var] = _calcWindDir(df_d['wspd_x_'+var.split('_')[1]],
                                    df_d['wspd_y_'+var.split('_')[1]])
             else:
-                logger.info(var,'in dataframe but not','wspd_x_'+var.split('_')[1],'wspd_x_'+var.split('_')[1])
+                logger.info(var+' in dataframe but not wspd_x_'+var.split('_')[1]+' nor wspd_x_'+var.split('_')[1])
```

Comment:
I don't understand this line

assume `var == wdir_u`
then `var.split('_')[1] == 'u'`

Then is the logged message:
`"wdir_u in dataframe but not wspd_x_u nor wspd_x_u"`

🤔

Reply:
`wdir` (u or l)  should not be averaged, instead `w_spd_x` and `w_spd_y` should be averaged separately and re-combined into an updated `wdir`
 These lines are meant to catch cases where `wdir` exists but not `w_spd_x` nor `w_spd_y`. In that case `wdir` cannot be averaged. It does happen sometime and I need to investigate why  `w_spd_x` and `w_spd_y` got lost.

Reply:
I was mostly puzzled by the log string. Shouldn't it be like this:

```suggestion
                logger.info(var+' in dataframe but not wspd_x_'+var.split('_')[1]+' nor wspd_y_'+var.split('_')[1])
```

Reply:
Yes you are right!